.gitlab-ci.yml

```
@@ -11,7 +11,7 @@ include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-golang-lint.yml'
   - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-golang-unittests.yml'
+    file: '.gitlab-ci-check-golang-unittests-v2.yml'
```

```suggestion
    file: '.gitlab-ci-check-golang-unittests-v2.yml'
    file: '.gitlab-ci-check-golang-unittests-v2-ssl.yml'
```
no strong opinions here, but the ssl will be needed in other places.